Modules/File/classes/Capabilities/Check/BaseCheck.php

```diff
@@ -38,17 +38,19 @@ protected function hasPermission(
         Context $context,
         Permissions ...$permission
     ): bool {
-        $permission_string = implode(
-            ',',
-            array_map(static fn(Permissions $permission) => $permission->value, $permission)
-        );
-        if ($context->getContext() === Context::CONTEXT_WORKSPACDE) {
-            return $helpers->workspace_access_handler->checkAccess(
-                $permission_string,
-                '',
-                $context->getCallingId(),
-                'file'
-            );
+        if ($context->getContext() === Context::CONTEXT_WORKSPACE) {
```

Comment:
As far as I can see the workspace access handler cannot handle a `,`-separated list of permissions. For the `Info` capability in the workspace context the permission string looked like this:

```
visible,read,view_content,read_learning_progress,edit_learning_progress,edit_permission,write,delete,copy,edit_file
```

An access check with this permission will always result in a boolean `false`.